var/spack/repos/builtin/packages/nfft/package.py

```diff
@@ -25,7 +25,7 @@
 from spack import *
 
 
-class Nfft(AutotoolsPackage):
+class Nfft(Package):
```

Comment:
Why is this being moved from `AutotoolsPackage` BACK to `Package`?  Without a good reason, that would seem to be the wrong direction.